ibis/expr/operations/relations.py

```diff
@@ -78,6 +78,67 @@ def resolve_name(self):
         return self.name
 
 
+@public
+class LocalTable(PhysicalTable):
```

Comment:
This looks very similar to `ops.UnboundTable`. Does that not work for your use case?

Reply:
`LocalTable` is used for reading from a local resource. Not related to UDT; please skip.